[New Product] Added Slackware GNU/Linux

alexkoster opened the pull request:
Adding Slackware as a new product!  Please feel free to modify or let me know what changes I can make.

alexkoster committed:
[slackware] New Product: Added Slackware

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+
+---
+
+> [Slackware](http://www.slackware.com/) is a general-purpose GNU/Linux distribution that aims to be most ["UNIX-like", simple and stable](http://www.slackware.com/info/).  The developers strive to minimize the number of changes to upstream software relases.  It is the oldest actively developed GNU/Linux distribution with development being led by creator Patrick Volkerding continuously since 1993 ...
```
```suggestion
> [Slackware](http://www.slackware.com/) is a general-purpose GNU/Linux distribution that aims to be most ["UNIX-like", simple and stable](http://www.slackware.com/info/). It strives to minimize the number of changes to upstream software releases.  It is the oldest actively developed GNU/Linux distribution with development being led by creator Patrick Volkerding continuously since 1993.
```

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+
+> [Slackware](http://www.slackware.com/) is a general-purpose GNU/Linux distribution that aims to be most ["UNIX-like", simple and stable](http://www.slackware.com/info/).  The developers strive to minimize the number of changes to upstream software relases.  It is the oldest actively developed GNU/Linux distribution with development being led by creator Patrick Volkerding continuously since 1993 ...
+
+There are several releases of Slackware Linux supported concurrently.  A [stability based release cycle](http://www.slackware.com/faq/do_faq.php?faq=general#4) is used and therefore releases are not due on a fixed schedule but have typically occured every few years.
```
```suggestion
There are several releases of Slackware Linux supported concurrently.  A [stability based release cycle](http://www.slackware.com/faq/do_faq.php?faq=general#4) is used and therefore releases are not due on a fixed schedule. A new release is usually made once every few years.
```

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+
+There are several releases of Slackware Linux supported concurrently.  A [stability based release cycle](http://www.slackware.com/faq/do_faq.php?faq=general#4) is used and therefore releases are not due on a fixed schedule but have typically occured every few years.
+
+Slackware's -current development tree closely follows bleeding edge of upstream releases.  The latest stable release gets bug and security fixes while previous non end of life (EOL) releases receive only security patches.
```
```suggestion
Slackware's `-current` development tree closely follows bleeding edge of upstream releases. The latest stable release gets bug and security fixes while previous non End-of-Life releases receive only security patches.
```

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+
+Slackware's -current development tree closely follows bleeding edge of upstream releases.  The latest stable release gets bug and security fixes while previous non end of life (EOL) releases receive only security patches.
+
+Slackware does not use systemd, instead using the traditional SysV for init (though extracted components from systemd such as [eudev](https://wiki.gentoo.org/wiki/Eudev) and [elogind](https://wiki.gentoo.org/wiki/Elogind) are present and utilized.)  Slackware officially supports 32-bit non-PAE (i586-compatible), 32-bit PAE (i686-compatible) and 64-bit x86 processors.  Unofficial builds are availa ...
```
```suggestion
Slackware officially supports 32-bit non-PAE (i586-compatible), 32-bit PAE (i686-compatible) and 64-bit x86 processors.  Unofficial builds are available for the arm and s/390 architectures.

Security Advisories are published [here](http://www.slackware.com/security/), and can be subscribed via the [slackware-security](http://www.slackware.com/lists/) mailing list.
```

Probably worth linking to the unofficial builds?

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+-   purl: pkg:os/slackwarelinux
+auto:
+-   distrowatch: slackware
+    regex: '^Distribution Release: Slackware Linux (?P<major>\d)\.(?P<minor>\d)$'
```
```suggestion
    regex: '^Distribution Release: Slackware (Linux )?(?P<major>\d+)\.(?P<minor>\d+)$'
```

https://regex101.com/r/Q5uFOI/1

Double digits, and older release didn't use `Linux` in the title

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+---
+permalink: /slackware
+alternate_urls:
+-   /slackwarelinux
```
```suggestion
-   /slackwarelinux
-   /slackware-linux
```

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2007-07-01
+-   releaseCycle: "11.0"
```
If there is a legitimate reason to keep these (they are really old releases!), lets remove `link` from all.

alexkoster replied:
@captn3m0 The reason I wanted to show old releases was to give users a sense of what a (typical) support lifecycle looks like for a Slackware stable release (since they often vary in time between release and EOL).  But I'm happy to remove them if you'd like.

usta replied:
@alexkoster i think he just means the `link` tags which are not shown when they reach eol dates
so adding those a bit useless plus almost half of them pointing wrong version.
One more thing when we use changelogTemplate thing we only add `link` tag if that release doesnt fit the changelogTemplate's one so as far as i can see you added link tag for each release which is not needed if you have
a working changelogTemplate

captn3m0 replied:
Removed incorrect `link` from old releases. I like the announce links better:
- http://www.slackware.com/announce/15.0.php vs https://mirrors.slackware.com/slackware/slackware-15.0/ChangeLog.txt

The former is definitely better. Going to update the changelogTemplate for the former and use that wherever possible.

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2002-06-19
+-   releaseCycle: "8.0"
+    eol: true
```
Can we guess a date here?

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
```
```suggestion
releaseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
```

typo

captn3m0 commented on products/slackware.md:
```diff
@@ -0,0 +1,109 @@
+    regex: '^Distribution Release: Slackware Linux (?P<major>\d)\.(?P<minor>\d)$'
+releases:
+-   releaseCycle: "current"
+    codename: "-current"
```
```suggestion
    codename: "-current"
    releaseLabel: Current
```

Avoids showing up as "current (-current)"

usta replied:
@captn3m0 do we need releaseLabel ? arent all we need is just remove codename tag here ?

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
Update products/slackware.md

Co-authored-by: Nemo <me@captnemo.in>

alexkoster committed:
[slackware] Adding Slackware 8.0 Release Cycle Est EOL

Estimating EOL date for Slackware 8.0 release cycle to be 203-03-29 from latest entry in ChangeLog.txt
ftp://mirror.cs.princeton.edu/pub/mirrors/slackware/slackware-8.0/ChangeLog.txt

captn3m0 committed:
Update slackware.md

captn3m0 committed:
Update slackware.md

captn3m0 committed:
Update slackware.md

captn3m0 reviewed (approved):
LGTM.
